backstop/src/storage.rs

```diff
@@ -329,21 +366,21 @@ pub fn set_reward_zone(e: &Env, reward_zone: &Vec<Address>) {
 ///
 /// ### Arguments
 /// * `pool` - The pool
-pub fn get_pool_eps(e: &Env, pool: &Address) -> i128 {
-    let key = BackstopDataKey::PoolEPS(pool.clone());
+pub fn get_pool_emissions(e: &Env, pool: &Address) -> i128 {
+    let key = BackstopDataKey::PoolEmis(pool.clone());
     get_persistent_default(e, &key, 0i128, LEDGER_THRESHOLD_SHARED, LEDGER_BUMP_SHARED)
 }
 
 /// Set the current emissions EPS the backstop is distributing to the pool
```

Comment:
There are a lot of references to EPS in the backstop that are confusing now as we don't store eps in the backstop anymore